sisu-osgi/sisu-osgi-connect/src/main/java/org/eclipse/sisu/osgi/connect/PlexusFrameworkConnectServiceFactory.java

```diff
@@ -238,11 +227,11 @@ private static void printFrameworkState(Framework framework, Logger log) {
 		Comparator<Bundle> bySymbolicName = Comparator.comparing(Bundle::getSymbolicName,
 				String.CASE_INSENSITIVE_ORDER);
 		Comparator<Bundle> byState = Comparator.comparingInt(Bundle::getState);
-		Arrays.stream(bundles).sorted(byState.thenComparing(bySymbolicName)).forEachOrdered(bundle -> {
-			log.info(toBundleState(bundle.getState()) + " | " + bundle.getSymbolicName() + " (" + bundle.getVersion()
-					+ ")");
+		Arrays.stream(bundles).sorted(byState.thenComparing(bySymbolicName)).forEachOrdered(b -> {
```

Comment:
can we keep `bundle` instead of `b` as a name here?

Reply:
Sure, done.